DEVELOPMENT.md

```diff
@@ -336,6 +336,25 @@ changes to existing stuff are required for a new feature.
 Prior to pgai v0.4.0, Python dependencies were installed system-wide. Until pgai versions 0.1 - 0.3 are deprecated
 [old dependencies](./src/old_requirements.txt) are installed system-wide.
 
+#### Release the extension
+
+1. Run `just ext clean-sql` in order to remove any prior generated dev sql.
+2. Edit the [build.py](projects/extension/build.py).
+   - Set the version to be released as the first element of the array returned by `versions` function. I.e. `"0.7.0",  # released`.
+3. Run `just ext build install`. This will generate, among other tasks, the final .sql files in `projects/extension/sql` and update the version in [__init__.py](projects/extension/ai/__init__.py).
```

Comment:
I don't think that we should be advocating for people to run `install` as part of the release process. It does a bunch of stuff which is not related to releasing, and which I would not want to do on my machine - for instance installing a ton of deps into the system Python.

Reply:
I run it in the development docker container. Since, the project directory is mounted in the development container, it "just works".
But fair point. I agree.

Reply:
Replaced with the just command @jgpruitt created recently `just ext build-release`.